dotCMS/src/main/java/com/dotcms/contenttype/model/field/FieldType.java

```diff
@@ -42,9 +44,14 @@ public String getHelpText() {
     }
 
     public Map<String, Object> toMap(){
+        final ImmutableList<String> propertiesName = properties.stream()
+                .map(property -> property.getName())
+                .collect(toImmutableList());
+
         return map("id", id,
```

Comment:
If the map do not suppose to change you can use imap witch is an immutable version of the map